src/scss/misc/fonts.scss

```diff
@@ -20,6 +20,14 @@ small {
   font-size: 80%;
 }
 
-a {
+a, .link-like {
```

Comment:
No, there are some pretty big conflicts with using `.link` and the idea is to style something like a link that may not actually be a link.